quickstarts/traefik/config.yml

```
@@ -0,0 +1,45 @@
+# Name of the quickstart (required)
+slug: traefik
+
+# Displayed in the UI (required)
+title: Traefik
+
+# Long-form description of the quickstart (required)
+description: |
+  The Traefik quickstart provides a dashboard that gives you a holistic view of the Traefik instances in your environment from telemetry collected through Prometheus Remote Write.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Easily install a curated dashboard to monitor the health of your Traefik instances.
+
+# Support level: New Relic | Verified | Community (required)
+level: Verified
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+  - Zack Mutchler
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - traefik
+  - prometheus
+  - remote write
+  - remote_write
+  - promql
+  - grafana
+
+documentation:
+  - name: Prometheus installation docs
+    description: |
+      Use Prometheus remote_write or New Relic's Prometheus OpenMetrics integration
+    url: https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIiwiZmF2b3JpdGVzIjp7InNlbGVjdGVkIjp0cnVlLCJ2aXNpYmxlIjp0cnVlfSwibGFzdFZpZXdlZCI6eyJzZWxlY3RlZCI6ZmFsc2UsInZpc2libGUiOnRydWV9fQ==&cards[0]=eyJuZXJkbGV0SWQiOiJwcm9tZXRoZXVzLXJlbW90ZS13cml0ZS1pbnRlZ3JhdGlvbi5ob21lIiwiYWNjb3VudElkIjoyMTM2Njc5fQ==&sidebars[0]=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5jYXRlZ29yaWVzIiwicm9vdE5lcmRsZXRJZCI6Im5yMS1jb3JlLmxpc3RpbmciLCJmYXZvcml0ZXMiOnsic2VsZWN0ZWQiOnRydWUsInZpc2libGUiOnRydWV9LCJsYXN0Vmlld2VkIjp7InNlbGVjdGVkIjpmYWxzZSwidmlzaWJsZSI6dHJ1ZX19&state=32a8b400-a43a-5a3d-3bc6-16621b554b08
```

If a new user is looking at this quickstart, they would not be able to navigate to this link without an account. Would you mind updating this URL to link to our Docs?

---

@josephgregoryii - I took this link from the pre-existing [prometheus-remote-write quickstart](https://github.com/newrelic/newrelic-quickstarts/blob/5e1b9f2306a685646c7dc757259c07ce80633d38/quickstarts/prometheus-remote-write/config.yml#L35-L39); do you think I should still shorten the URL to the public docs site equivalent?

---

That link takes you directly to the in-product install experience for remote-write. Could we update the link to use our official documentation?
```suggestion
    url: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/
```
We should probably get the other one updated as well

---

understood; changes committed, thanks for the feedback!